src/analysis/ConfigurationsExt.cpp

```diff
@@ -257,16 +275,26 @@ namespace espressopp {
              int index = ids[i];
 
              LOG4ESPP_INFO(logger, "set coordianates of particle with id = " << index <<
-                                   ": " << coordinates[3*i] << " " <<  coordinates[3*i+1] << " " << coordinates[3*i+2] << 
-                                   "and velocities: " <<  velocities[3*i] << " " <<  velocities[3*i+1] << " " << velocities[3*i+2]);
+                                   ": " << coordinates[3*i] << " " <<  coordinates[3*i+1] << " " << coordinates[3*i+2] <<
+                                   "and velocities: " <<  velocities[3*i] << " " <<  velocities[3*i+1] << " " << velocities[3*i+2] <<
+                                   "and types: " <<  types[i]);
+
+
+             //RealND _vec(6);
+             RealND _vec(7);  // p[0].p[1].p[2].v[0].v[1].v[2].type
 
+             for (int k=0; k<1; k++)
+               _vec.setItem(k, types[i]);
```

Comment:
No, makes no sense. I thought of that and then I kept the loop...will remove it!